routers/web/web.go

```diff
@@ -352,8 +354,13 @@ func RegisterRoutes(m *web.Route) {
 					Post(bindIgnErr(forms.SignUpOpenIDForm{}), auth.RegisterOpenIDPost)
 			}, openIDSignUpEnabled)
 		}, openIDSignInEnabled)
-		m.Get("/sign_up", auth.SignUp)
-		m.Post("/sign_up", bindIgnErr(forms.RegisterForm{}), auth.SignUpPost)
+
+		m.Group("/sign_up", func() {
+			m.Use(httprate.LimitByIP(10, 1*time.Minute))
```

Comment:
```suggestion
			m.Use(httprate.LimitByRealIP(10, 1*time.Minute))
```

LimitByIP only goes by RemoteAddr, which won't work behind reverse proxies.

Reply:
Thought, `KeyByRealIP` is also flawed because it it vulnerable to XFF spoofing, so idk, maybe there is a better module around.